helpers/build_packages.sh

```diff
@@ -132,6 +132,8 @@ if [ "$BUILDMW_REPO" == "" ]; then
 # hack until upstream is sane
 if (grep -q 'PLATFORM_VERSION := 6.' $ANDROID_ROOT/build/core/version_defaults.mk); then
 buildmw libhybris mm64-rpm || die
+elif (grep -q 'PLATFORM_VERSION := 7.' $ANDROID_ROOT/build/core/version_defaults.mk); then
+buildmw libhybris n64-rpm || die
 else
 buildmw libhybris || die
 fi
```

Comment:
down below there is this line:
``buildmw geoclue-providers-hybris jb36857 || die``
I think it's safe to move it back to master? Android 7 certainly needs the latest master